ui/app/pages/swaps/awaiting-swap/awaiting-swap.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+
+import {
+  renderWithProvider,
+  createSwapsMockStore,
+} from '../../../../../test/jest';
+import AwaitingSwap from '.';
+
+describe('AwaitingSwap', () => {
+  const createProps = (customProps = {}) => {
+    return {
+      swapComplete: false,
+      txHash: 'txHash',
+      tokensReceived: 'tokensReceived',
+      submittingSwap: true,
+      inputValue: 5,
+      maxSlippage: 3,
+      ...customProps,
+    };
+  };
+  const store = configureMockStore()(createSwapsMockStore());
+
+  it('renders the component with initial props', () => {
+    const { container, getByText } = renderWithProvider(
+      <AwaitingSwap {...createProps()} />,
+      store,
+    );
+    expect(getByText('[swapProcessing]')).toBeInTheDocument();
```

Comment:
This [swapProcessing] string should be the actual text value we'd expect to see, imo. The change mentioned in the localization mock will make that a possibility.

Reply:
Absolutely agree. I have this on my todo list to make sure we use real translations instead of keys in our unit tests. I will try to do that as part of this PR.